src/App.tsx

```diff
@@ -43,6 +44,9 @@ const App = () => {
             <Route path="/speed-send/add" exact component={MessageTemplate} />
             <Route path="/speed-send/:id/edit" exact component={MessageTemplate} />
             <Route path="/chat" exact component={Chat} />
+            <Route path="/templates" exact component={HSMPage} />
+            {/* This part isn't working properly */}
```

Comment:
Why is this added? Please clean up unnecessary code.

Reply:
That comment was there before, I wasn't sure who added it